examples/03-Maxwell/Maxwell3DTeam7.py

```diff
@@ -345,98 +351,79 @@
     [-1.35, -0.71, -0.81, -0.67, 0.15, 1.39, 2.67, 3.00, 4.01, 3.80, 4.00, 3.02, 2.20, 2.78, 1.58, 1.37, 0.93],
 ]
 
+################################################################################
+# Write dataset values in a .csv file
```

Comment:
```suggestion
# Write dataset values in a CSV file
```
File extensions should be written in all capital letters (with no preceding period).